### PR TITLE
34 Get Appointment by Appointment ID

### LINKED ISSUE #34: Get an appointment for a notary user
As a user I can get an appointment for a notary user by sending a GET request to the `/api/v1/user/:notary_user_id/appointments/:appointment_id` endpoint.

As a user I can expect a successful response of:
```
Status: 200

 {
    "id": 1,
    "notary": {
      "name": "William Warrick",
      "id": 2
    },
    "appointee": {
      "name": "Nancy Noteralli",
      "id": 1
    },
    "status": "Pending",
    "time": "23:15:42",
    "date": "2020-02-20",
    "location": "Irving, TX, USA"
  }
```

### PR DESCRIPTION
**Pull request by BabsLabs**
## Type of change
- [x] :hatching_chick: New feature (non-breaking change which adds functionality)
- [x] :page_with_curl: This change requires a documentation update

## Summary of changes
- Add endpoint for a `GET` request to `/api/v1/notaries/:notary_id/appointments/:appointment_id`

## How Has This Been Tested?
- [x] :black_square_button: Integration testing

## Checklist:
- [x] :white_check_mark: I have performed a self-review of my own code
- [x] :page_facing_up: I have made corresponding changes to the documentation
- [x] :no_entry_sign: My changes generate no new warnings
- [x] :heavy_check_mark: I have added tests that prove my fix is effective or that my feature works
- [x] :100: New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +157,30 @@ Successful response should look like:
 ]
 ```
 
+###### Get An Appointment By Id
```
Comment (lrs8810) [inline README.md:160]: Great job keeping the ReadMe up-to-date!

File: notary_now_api/tests.py [hunk 2]
```diff
@@ -144,3 +144,58 @@ def test_appointments_endpoint(self):
...
+        self.assertEqual(json_response['time'], self.appointment_one.time)
+        self.assertEqual(json_response['status'], self.appointment_one.get_appointment_result)
+
+        self.assertNotEqual(json_response['id'], self.appointment_two.id)
+        self.assertNotEqual(json_response['time'], self.appointment_two.time)
+
+    def test_appointment_endpoint_sad_path(self):
+        request = self.factory.get(f'/api/v1/notaries/20/appointments/{self.appointment_one.id}')
+        response = appointment_detail(request, 20, self.appointment_one.id)
+        self.assertEqual(response.status_code, 400)
```
Comment (davidlopez) [inline notary_now_api/tests.py:201]: Sadpath tests!! Great Job

File: notary_now_api/tests.py [hunk 3]
```diff
@@ -144,3 +144,58 @@ def test_appointments_endpoint(self):
...
+        self.assertEqual(json_response['appointee']['name'], self.appointee.first_name + " " + self.appointee.last_name)
+        self.assertEqual(json_response['location'], self.appointment_one.location)
+        self.assertEqual(json_response['date'], self.appointment_one.date)
+        self.assertEqual(json_response['time'], self.appointment_one.time)
+        self.assertEqual(json_response['status'], self.appointment_one.get_appointment_result)
+
+        self.assertNotEqual(json_response['id'], self.appointment_two.id)
+        self.assertNotEqual(json_response['time'], self.appointment_two.time)
+
+    def test_appointment_endpoint_sad_path(self):
```
Comment (lrs8810) [inline notary_now_api/tests.py:198]: Great TDDing! Also, great job testing the sad path

File: notary_now_api/views.py [hunk 4]
```diff
@@ -73,4 +73,11 @@ def appointments(request, pk):
...
     else:
-        return JsonResponse({'error': 'Notary not Found'}, status=404)
+        return JsonResponse({'error': 'Notary Not Found'}, status=404)
+
+def appointment_detail(request, notary_id, appointment_id):
+    appointment = Appointment.objects.filter(notary_id=notary_id, id=appointment_id)
+    if appointment:
+        return JsonResponse(format_appointment(appointment[0]), status=200, safe=False)
+    else:
+        return JsonResponse({'error': 'Notary Appointment Relation Not Found'}, status=400)
```
Comment (davidlopez) [inline notary_now_api/views.py:83]: Made that look easy

**Review by davidlopez — APPROVED** [review]

**Review by lrs8810 — APPROVED** [review]

**Pull request merged**